README.md

```diff
@@ -3,32 +3,37 @@
 This project shows an example of a multi-llm multimedia enabled chatbot that can do web research. This example supports accessing
 the researcher agent either directly, using REST, or through a chatbot interface
 
-## Core Concepts
-1. Customizing the AgentProcessingUnit
-2. Running the UI
+## Agents
+### research_agent
+The `research_agent` is the top level agent that coordinates web research using the search_agent and scraping_agent.
 
-## Directory Structure
+### search_agent
+Can perform free form research on the web using a Google Custom Search Engine.
```

Comment:
It looks like CSE has been superceeded by PSE: https://chatgpt.com/share/6703fb44-8094-8000-9108-5cd64134503a.

Also, is the search agent actually doing "research" or just returning a list of search results that can then be scraped by the scraping_agent and interpreted by the LLM? 

If so, perhaps change this line to:

`Returns web search results using <a href="https://developers.google.com/custom-search" target=_blank>Google Programmable Search Engine</a> (formerly Custom Search Engine, or CSE).`